constants.go

```diff
@@ -32,6 +32,8 @@ const (
 	opClose           = -11
 	opSetAuth         = 100
 	opSetWatches      = 101
+	opSetWatches2     = 105
```

Comment:
what's up with values 102-104?

Reply:
theres a bunch of ops that the client doesnt support yet in the zk protocol